mappings/net/minecraft/util/collection/LowercaseMap.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_3522 net/minecraft/util/collection/LowercaseMap
```

Comment:
`Lowercased` I suggest

Reply:
?

I'm not changing the class name, just moving it.